react/react2/week1/github-search/src/components/InputBar.js

```diff
@@ -0,0 +1,22 @@
+import { useContext } from "react";
+import { SearchContext } from "./SearchContext";
+
+export const InputBar = () => {
+  const contextValue = useContext(SearchContext);
+
+  const inputHandler = (e) => {
+    // console.log("Input handler works");
+    // console.log(contextValue.inputValue);
+    contextValue.setInputValue(e.target.value);
+  };
+
+  return (
+    <div className="inputBar">
+      <input
+        type="text"
+        onChange={inputHandler}
+        placeholder="Search for user"
+      ></input>
+    </div>
```

Comment:
This file name coulde be a better one, for Ex: searchBar, UserSearch, etc.

Reply:
Good point - gonna fix it :)